typescript/examples/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "reality-capture-examples",
+	"name": "@itwin/reality-capture-examples",
```

Comment:
not publishing this

Reply:
added itwin prefix to sdk and sdk-node, the two to be published, added private to the rest